extensions/json-language-features/server/src/jsonServerMain.ts

```diff
@@ -34,6 +34,10 @@ namespace SchemaContentChangeNotification {
 	export const type: NotificationType<string, any> = new NotificationType('json/schemaContent');
 }
 
+namespace SchemaRetryNotification {
+	export const type: NotificationType<string, any> = new NotificationType('json/schemaRetry');
```

Comment:
I would call the request `json/validate` and even return diagnostics. It's something we will need anyways in the future to allow programmatic validation